source/patterns/@aws-solutions-constructs/aws-eventbridge-lambda/README.md

```diff
@@ -75,7 +78,7 @@ _Parameters_
 Out of the box implementation of the Construct without any override will set the following defaults:
 
 ### Amazon EventBridge Rule
-* Grant least privilege permissions to EventBridge rule to trigger the Lambda Function
+* Grant least privilege permissions to EventBridge rule created on `Default` EventBus to trigger the Lambda Function
```

Comment:
The ** Default Settings ** section reads `Out of the box implementation of the Construct without any override will set the following defaults`, I thought it would be more appropriate to call out `default` EventBus is assumed when construct is used with it's defaults.

Reply:
As you've currently changed it, it is factually inaccurate whenever someone takes advantage of the new functionality you are adding.

Reply:
Not corrected